alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/binder/solr/sharding/SolrShardingMetrics.java

```diff
@@ -54,6 +54,15 @@ public void updateMetrics() {
                 shardInstances.forEach(shardInstance -> {
                     Tags instanceTags = shardTags.and(Tags.of("instanceHost", shardInstance.getHostName()));
                     ShardState shardState = solrShardingMetricsContainer.getShardState(shardInstance);
+                    //If shardstate doesn't exist, return all -1
+                    if(shardState == null){
+                        shardState.setLastIndexedChangeSetId(-1);
```

Comment:
NullPointer garantee?

Reply:
Yeah I did a rebase to fix this but Github doesn't seem to update.